featurebyte/models/credential.py

```diff
@@ -73,6 +73,15 @@ def encrypt(self) -> None:
         for field in self.__fields__.values():
             if field.type_ == StrictStr:
                 setattr(self, field.name, encrypt_value(getattr(self, field.name)))
+            elif field.type_ == str:
```

Comment:
Is this a new behaviour and is it covered in tests?

Reply:
This is covered by the credential model unit tests for encryption functionality. Without this change the tests that use GCS storage credential will fail

Reply:
Is it possible to have a unit test for this?

Reply:
There is already a test. It was not covered previously because GCS credential is not in the test parameters but that is updated in this PR

Reply:
Sorry, I missed that!